packages/core/src/Modules/Cashier/Containers/cashier.jsx

```diff
@@ -57,14 +49,23 @@ class Cashier extends React.Component {
             return options;
         };
 
+        const { routes: routes_config, location } = this.props;
+        const selected_route = isMobile()
+            ? getSelectedRoute({ routes: routes_config, pathname: location.pathname })
```

Comment:
Why was this moved to the render block?

Reply:
This is to reuse `getSelectedRoute` logic and to use `selected_route.component` as a jsx Tag